ColorPicker/ColorPickerSlider.js

```diff
@@ -0,0 +1,525 @@
+ * @memberof sandstone/ColorPicker
+ * @ui
+ * @private
+ */
+const ColorPickerSliderRGB = ({disabled, selectedColor, selectedColorHandler, ...props}) => {
+	const {red, green, blue} = hexToRGB(selectedColor);
+	const [localRed, setLocalRed] = useState(red);
+	const [localGreen, setLocalGreen] = useState(green);
+	const [localBlue, setLocalBlue] = useState(blue);
+
+	useEffect(() => {
+		setLocalRed(red);
+		setLocalGreen(green);
+		setLocalBlue(blue);
+	}, [blue, green, red]);
+
+	const changeValueRed = useCallback((ev) => {
+		setLocalRed(ev.value);
+	}, []);
+
+	const changeValueGreen = useCallback((ev) => {
+		setLocalGreen(ev.value);
+	}, []);
+
+	const changeValueBlue = useCallback((ev) => {
+		setLocalBlue(ev.value);
+	}, []);
+
+	const changeSelectedColor = useCallback(() => {
+		selectedColorHandler(rgbObjectToHex({red: localRed, green: localGreen, blue: localBlue}));
+	}, [localBlue, localGreen, localRed, selectedColorHandler]);
+
+	return (
+		<Layout {...props} className={componentCss.slidersContainer}>
+			<Cell>
+				<Cell className={componentCss.labelText}>Red</Cell>
+				<Row className={componentCss.sliderRow}>
+					<Cell
+						className={componentCss.sliderCell}
+						size="70%"
+						style={{backgroundImage: `linear-gradient(to right, rgb(0,${localGreen},${localBlue}), rgb(255,${localGreen},${localBlue}))`}}
+					>
+						<Slider
+							activateOnSelect
+							css={componentCss}
+							disabled={disabled}
+							max={255}
+							min={0}
+							noFill
+							onBlur={changeSelectedColor}
+							onChange={changeValueRed}
+							onKeyUp={changeSelectedColor}
+							onPointerUp={changeSelectedColor}
+							spotlightDisabled={disabled}
+							style={{
+								'--sand-slider-knob-border-color': generateOppositeColor(rgbObjectToHex({red: localRed, green: localGreen, blue: localBlue})),
+								'--sand-focus-bg-color-rgb': `${localRed},${localGreen},${localBlue}`,
+								'--sand-progress-slider-color': rgbObjectToHex({red: localRed, green: localGreen, blue: localBlue})
+							}}
+							value={localRed}
+						/>
+					</Cell>
+					<Cell className={componentCss.outputText} size="20%">{localRed}</Cell>
+				</Row>
+				<Cell className={componentCss.labelText}>Green</Cell>
+				<Row className={componentCss.sliderRow}>
+					<Cell
+						className={componentCss.sliderCell}
+						size="70%"
+						style={{backgroundImage: `linear-gradient(to right, rgb(${localRed},0,${localBlue}), rgb(${localRed},255,${localBlue}))`}}
+					>
+						<Slider
+							activateOnSelect
+							css={componentCss}
+							disabled={disabled}
+							max={255}
+							min={0}
+							noFill
+							onBlur={changeSelectedColor}
+							onChange={changeValueGreen}
+							onKeyUp={changeSelectedColor}
+							onPointerUp={changeSelectedColor}
+							spotlightDisabled={disabled}
+							style={{
+								'--sand-slider-knob-border-color': generateOppositeColor(rgbObjectToHex({red: localRed, green: localGreen, blue: localBlue})),
```

Comment:
@MikyungKim it's because '--sand-slider-knob-border-color' CSS variable was needed here to change the border dynamically. Based on the color the knob's border changes between white and black. 
If we don't want to touch that variable that was created long ago, I could create a separate one for @sand-colorpicker-slider-knob-border-color. 
Do you think it's better to do this?

Reply:
Ah - I see. Let's just merge as-is.